__tests__/commands/openapi/validate.test.ts

```diff
@@ -1,37 +1,25 @@
-/* eslint-disable no-console */
-
 import fs from 'node:fs';
 
-import chalk from 'chalk';
 import prompts from 'prompts';
-import { describe, beforeAll, beforeEach, afterEach, it, expect, vi, type MockInstance } from 'vitest';
+import { describe, beforeAll, beforeEach, afterEach, it, expect, vi } from 'vitest';
 
 import Command from '../../../src/commands/openapi/validate.js';
 import { after, before } from '../../helpers/get-gha-setup.js';
-import { runCommandAndReturnResult, runCommandWithHooks } from '../../helpers/oclif.js';
-
-let consoleInfoSpy: MockInstance<typeof console.info>;
-
-const getCommandOutput = () => {
-  return [consoleInfoSpy.mock.calls.join('\n\n')].filter(Boolean).join('\n\n');
-};
+import { runCommand, runCommandWithHooks, type OclifOutput } from '../../helpers/oclif.js';
 
 describe('rdme openapi validate', () => {
-  let run: (args?: string[]) => Promise<string>;
+  let run: (args?: string[]) => OclifOutput;
   let testWorkingDir: string;
 
   beforeAll(() => {
-    run = runCommandAndReturnResult(Command);
+    run = runCommand(Command);
   });
 
   beforeEach(() => {
-    consoleInfoSpy = vi.spyOn(console, 'info').mockImplementation(() => {});
```

Comment:
we were previously handrolling mocks for `console.info` in addition to the console output mocking we get via `@oclif/test`. what a mess